presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/ColumnStatistics.java

```diff
@@ -57,6 +57,46 @@ public long getNumberOfValues()
         return numberOfValues == null ? 0 : numberOfValues;
     }
 
+    public boolean hasMinAverageValueSizeInBytes()
+    {
+        return hasNumberOfValues() && numberOfValues > 0;
+    }
+
+    /**
+     * The minimum average value sizes.
+     * The actual average value size is no less than the return value.
+     * It provides a lower bound of the size of data to be loaded
+     */
+    public long getMinAverageValueSizeInBytes()
+    {
+        if (!hasMinAverageValueSizeInBytes()) {
+            return 0L;
+        }
+        if (booleanStatistics != null) {
+            return 1L;
+        }
+        if (integerStatistics != null) {
+            return Integer.BYTES;
+        }
+        if (doubleStatistics != null) {
+            return Double.BYTES;
+        }
+        if (stringStatistics != null) {
+            return stringStatistics.getSum() / numberOfValues;
```

Comment:
I think we should add in `4` bytes to string and binary to account for the length int.  This could become significant for really wide tables 1-10k streams.  Also, we should add in an extra byte for the `null` field.